test/unit/specs/components/geocoder/Geocoder.spec.js

```diff
@@ -0,0 +1,35 @@
+import { shallowMount } from '@vue/test-utils';
+import Geocoder from '@/components/geocoder/Geocoder';
+
+describe('geocoder/Geocoder.vue', () => {
+  // Inspect the raw component options
+  it('is defined', () => {
+    expect(typeof Geocoder).to.not.equal('undefined');
+  });
+
+  describe('props', () => {
+    let comp;
+    beforeEach(() => {
+      comp = shallowMount(Geocoder);
```

Comment:
Please look at `test/unit/specs/components/ol/Map.spec.js` or `test/unit/specs/components/measuretool/MeasureWin.spec.js` how to inject an app-config for the tests.

Reply:
Took some time (and sweat) but added more unit tests, in particular invoking the search input and combobox and verifying results even on map. Still could not get fake XHR and Timers working with `Sinon`, so faked with a `setTimeout()` in the test. So a real  "Nominatim" search is performed for a well-known place/address in Bonn...

Also merged with latest version upgrades from #104 like OL6, Vue, Vuetify, and still works.

Made a slide implementation change: replacing callback from the `GeocoderController`to the `Geocoder` with a real `async/await` approach. Made testing also easier.

So unless more tests are required this is done, possibly additional issues for enhancements.